### ISSUE TITLE
Shapeshifter shows many duplicates of a copied selection

### ISSUE THREAD

**Issue by giubi** [ffMathy/Shapeshifter#597]
**Description**
With Shapeshifter 5, I used to select and copy text and, regardless of how long the pression of CTRL+C lasted or how many times I pressed the same combination, the selection was copied only once. With v6, Shapeshifter unnecessarily shows many duplicates of the same selection, which greatly increases the list of copied items, adding a lot of confusion, especially when you have a large amount of content in your clipboard.

Thanks in advance. A long time Shapeshifter user.

**Comment by ffMathy** [comment]
Shapeshifter is currently quite dormant unfortunately. I'm the sole maintainer, doing this for fun, and I have [many other open source projects of my making](https://github.com/ffMathy?tab=repositories) to attend to.

If you'd like to show your support and potentially accelerate the fix of this issue, consider becoming a sponsor on Patreon. That way, you don't actually pay anything unless major updates arrive. And I don't expect them to arrive until quite some time, but if enough people become sponsors, I'll probably start doing this as my main project.

I hope you understand!
